ts_sdk/cli/__put_cmd.py

```diff
@@ -61,15 +69,42 @@ def __ensure_args(args: argparse.Namespace):
     args.ignore_ssl = args.ignore_ssl in [True, 'true', 'True', '1']
 
 
+def __validate_ids(args):
+    ids_dir = args.folder
+
+    ids_validator_process = subprocess.run(
+        ["python", "-m", "ids_validator", ids_dir],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
```

Comment:
Since you're not using stderr anymore, you can probably get rid of this.  However, I think it'd be better to include that in the output.  Maybe using [subprocess.STDOUT](https://docs.python.org/3/library/subprocess.html#subprocess.STDOUT) so stdout and stderr are interleaved, making it easier to see when stderr output occurred:
```suggestion
        stderr=subprocess.STDOUT,
```

Reply:
Subprocess is not called any more. 
Instead validate_ids fx will be called